front-end/src/components/views/Chart.js

```diff
@@ -130,7 +130,10 @@ const Chart = ({ chartInformation, video, videoTimestamp }) => {
       });
       // Updates the chart to show the point that is closest to the video timestamp
       if (seriesPointIndeces.length === 0) return;
-      chartRef.current.series[seriesPointIndeces[0].series].points[seriesPointIndeces[0].point].onMouseOver();
+      const point = chartRef.current.series[seriesPointIndeces[0].series].points[seriesPointIndeces[0].point];
+      // Update chart options so that point.x is the value of the first plotline
+      setChartOptions(movePlotLine(chartOptions, point.x));
+      point.onMouseOver();
```

Comment:
I tried commenting out this line and I really like the simplicity of just the bar, what do you think? Imo we should look into displaying the value somewhere else, this could also get you multiple series' values all at once which would be a huge help / epic feature for other purposes too.

Reply:
removed it